src/feed-leaf.tsx

```diff
@@ -13,17 +14,27 @@ export interface Properties {
   animationUrl?: string;
   minter?: string;
   owner?: string;
-  attributes?: { trait_type: string, value: string }[];
+  attributes?: { trait_type: string; value: string }[];
   ipfsContentId: string;
   metadataUrl: string;
   chainId: number;
   contract: string;
+  znsRoute?: string;
 }
 
 export class FeedLeaf extends React.Component<Properties, {}> {
   onCopy = (text: string): void => {
     navigator.clipboard.writeText(text);
   }
+  shareNFT = (): void => {
+    const { znsRoute } = this.props;
+
+    window.open(
+      'https://twitter.com/intent/tweet?url=https://share.market.wilderworld.com/'+znsRoute?.split('wilder.')[1],
```

Comment:
eg. at the very least we should be testing that this is called in the correct way, based on the znsRoute that is provided.

Reply:
yup! the nice thing about react is that it gives an easy way of defining a public interface allowing you to just pass in anything you require. so, you can just pass window in through the props. (also, let's use the global package for accessing window, rather than accessing directly. `import window from 'global/window';`) i can show you more specifically, if you'd like.
*however*
is there a reason we are using window.open and not just a link?

Reply:
i think using a link makes more sense for something like this, unless there's a specific reason not to.

Reply:
for curiosity sake, there are two main ways to inject things like this:
- using an HOC: https://github.com/zer0-os/zOS/blob/development/src/lib/web3/provider-service.tsx#L16
- or just exporting a wrapper that does the injection: https://github.com/zer0-os/zOS/blob/development/src/components/address-bar/container.tsx#L113

the first is better for things that will be used by many components, the second is fine for one-offs.